test/Performance/setup_env_unix.sh

```diff
@@ -1,140 +1,184 @@
+    printf "done\n"
+
+    printf "Installing gcc, git, zip libxml, openssl, EPEL, python3, pip3..."
+    yes | sudo yum install -y gcc-c++ libxml2-devel git zip openssl-devel python34 python34-devel python34-pip >> env_setup.log
+    printf "done\n"
+
+    printf "Installing MSODBCSQL..."
     curl -s https://packages.microsoft.com/config/rhel/7/prod.repo > /etc/yum.repos.d/mssql-release.repo
-    (yes | sudo ACCEPT_EULA=Y yum install -y msodbcsql >> env_setup.log 2>&1)
-    (yes | sudo yum install -y unixODBC-devel autoconf >> env_setup.log 2>&1)
-    echo "OK"
-    echo "Installing pyodbc"
-    pip3 install --upgrade pip >> env_setup.log 2>&1
-    pip3 install pyodbc >> env_setup.log 2>&1
-    echo "OK"
+    (yes | sudo ACCEPT_EULA=Y yum install -y msodbcsql >> env_setup.log)
+    (yes | sudo yum install -y unixODBC-devel autoconf >> env_setup.log)
+    printf "done\n"
+
+    printf "Installing pyodbc"
+    pip3 install --upgrade pip >> env_setup.log
+    pip3 install pyodbc >> env_setup.log
+    printf "done\n"
+
 elif [ $PLATFORM = "Sierra" ]; then
-    echo "Installing homebrew..."
-    yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing wget..."
-    brew install wget >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing svn..."
-    brew install svn >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing openssl..."
-    brew install pkg-config >> env_setup.log 2>&1
-    brew install openssl >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing python3..."
-    brew install python3 >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing MSODBCSQL..."
-    brew tap microsoft/msodbcsql https://github.com/Microsoft/homebrew-msodbcsql >> env_setup.log 2>&1
-    brew update >> env_setup.log 2>&1
-    yes | ACCEPT_EULA=Y brew install msodbcsql >> env_setup.log 2>&1
-    echo "OK"
-    yes | brew install autoconf >> env_setup.log 2>&1
-    echo "Installing pyodbc..."
-    pip3 install pyodbc >> env_setup.log 2>&1
-    echo "OK"
+    printf "Installing homebrew..."
+    yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" >> env_setup.log
+    printf "done\n"
+
+    printf "Installing wget..."
+    brew install wget >> env_setup.log
+    printf "done\n"
+
+    printf "Installing svn..."
+    brew install svn >> env_setup.log
+    printf "done\n"
+
+    printf "Installing openssl..."
+    brew install pkg-config >> env_setup.log
+    brew install openssl >> env_setup.log
+    printf "done\n"
+
+    printf "Installing python3..."
+    brew install python3 >> env_setup.log
+    printf "done\n"
+
+    printf "Installing MSODBCSQL..."
+    brew tap microsoft/msodbcsql https://github.com/Microsoft/homebrew-msodbcsql >> env_setup.log
+    brew update >> env_setup.log
+    yes | ACCEPT_EULA=Y brew install msodbcsql >> env_setup.log
```

Comment:
there's a homebrew bug in installing ODBC now. Need to change this line to 'brew install --no-sandbox msodbcsql' (I noticed this when I was tesing)